### PR TITLE
adjust padding in Portfolio and Trade components

### PR DESCRIPTION
**Pull request by AndresImach**

### CODE REVIEW

File: src/pages/Trade/trade-styles.ts [hunk 1]
```diff
@@ -10,7 +10,22 @@ export const StyledFlex = styled(Flex)`
...
 export const TradeContainer = styled(Flex)`
   @media (min-width: ${theme.breakpoints.xxl}) {
-    padding-left: 60px;
-    padding-right: 28px;
+    padding-left: 16px;
   }
-`
+`;
+
+export const LineSeparator = styled(Flex)`
```
Comment (Aliona-D) [inline src/pages/Trade/trade-styles.ts:17]: it is not used anywhere,  is it?

Comment (AndresImach) [reply src/pages/Trade/trade-styles.ts:17]: I see that it's being used on PositionsTable in Trade page.

Comment (Aliona-D) [reply src/pages/Trade/trade-styles.ts:17]: yes, it is used on Positions table, but it is taken from positions-table-styles.ts. And on Trade page it is not used

Comment (AndresImach) [reply src/pages/Trade/trade-styles.ts:17]: oh ok. We can get reed of it then. ty

**Review by Aliona-D — APPROVED** [review]

**Pull request merged**